rudder-agent/SPECS/rudder-agent.spec

```diff
@@ -281,6 +294,12 @@ BuildRequires: curl-devel
 Requires: curl
 %endif
 
+## JQ dependencies
+%if "%{use_system_jq}" == "true"
+BuildRequires: jq-devel
+Requires: jq
+%endif
+
```

Comment:
I think you forgot to use the use_system_jq variable in the makes commands of this file